src/app/user-page/user-page.component.html

```diff
@@ -0,0 +1,132 @@
+      <span><mat-icon>people</mat-icon></span>
+      <div>{{ user?.username }}</div>
+    </div>
+  </div>
+</app-header>
+
+<div class="container">
+  <div fxLayout fxLayout.lt-md="column" fxLayoutAlign="center" fxLayoutGap="1rem" class="my-5">
+    <mat-card fxFlex.gt-md="25" fxFlex.lt-md="100" fxLayout="column" fxLayoutAlign="center">
+      <img mat-card-image [src]="user?.avatarUrl" class="mt-0 mx-auto mb-3" />
+      <mat-card-content fxLayout="column" fxLayoutAlign="center center">
+        <h4 matLine class="accent-1-dark mt-2 mb-0">{{ user?.username }}</h4>
+      </mat-card-content>
+    </mat-card>
+
+    <div fxLayout="column" fxLayoutGap="1rem" fxFlex>
+      <!-- GitHub -->
+      <mat-card *ngIf="gitHubProfile" fxLayout="column" fxLayoutAlign="space-between" class="h-100">
+        <div fxLayout fxLayoutAlign="space-between">
+          <mat-card-header>
+            <div mat-card-avatar matBadge="✓" class="mr-2">
+              <img src="../../assets/images/account-logos/github.svg" alt="Github logo" class="w-100" />
+            </div>
+            <mat-card-title class="normal-line-height">GitHub Account</mat-card-title>
+            <mat-card-subtitle class="normal-line-height weight-bold accent-1-dark mt-2 mb-0" fxLayout.lt-md="column" fxLayoutGap="1rem">
+              <span *ngIf="gitHubProfile.name"
+                >Account Name: <span>{{ gitHubProfile?.name }}</span></span
+              >
+              <span>
+                Username:
+                <a href="https://github.com/{{ gitHubProfile?.username }}" target="_blank" class="accent-1-dark">
+                  <span>{{ gitHubProfile?.username }}</span>
+                </a>
+              </span>
+            </mat-card-subtitle>
+          </mat-card-header>
+          <span>
+            <img [src]="gitHubProfile.avatarURL" alt="GitHub avatar" />
+          </span>
+        </div>
+        <div *ngIf="!user?.orcid && !googleProfile" fxLayout="column" fxLayoutAlign="end" class="mt-3">
+          <hr />
+          <span fxLayout fxLayoutAlign="end end">
+            <a href="https://github.com/{{ gitHubProfile?.username }}" target="_blank">
+              <button mat-button class="accent-1-dark-btn small-mat-btn-skin small-btn-structure mt-2">
+                <mat-icon class="mat-icon-sm">open_in_new</mat-icon>
+                View
+              </button>
+            </a>
+          </span>
+        </div>
+      </mat-card>
+      <!-- ORCID -->
+      <mat-card *ngIf="user?.orcid" class="h-100">
+        <mat-card-header>
+          <div mat-card-avatar matBadge="✓" class="mr-2">
+            <img src="../../assets/images/account-logos/orcid.svg" alt="ORCID logo" class="w-100" />
+          </div>
+          <mat-card-title class="normal-line-height">ORCID Account</mat-card-title>
+          <mat-card-subtitle class="accent-1-dark normal-line-height mt-2 mb-0">
+            <a href="https://orcid.org/{{ user?.orcid }}" target="_blank">
```

Comment:
Last comment on a redundant question mark (skipped some above), and they don't hurt, but you know user has to exist because of line 57.